tobac/utils/internal.py

```diff
@@ -442,7 +442,13 @@ def njit_if_available(func, **kwargs):
         from numba import njit
 
         return njit(func, kwargs)
-    except ModuleNotFoundError:
+    except KeyboardInterrupt as kie:
+        raise
```

Comment:
Should this be `raise kie` or will it raise the keyboard exception implicitly?

Reply:
This is intentional; if you `raise kie`, you lose the stack trace. See also https://docs.python.org/3/tutorial/errors.html#raising-exceptions

Reply:
Ah great, I didn't know that!